src/state.rs

```diff
@@ -24,7 +24,49 @@ pub struct CallFrame {
     // to support in-memory vs on-disk storage, etc.
     pub storage: HashMap<U256, U256>,
 }
-
+#[derive(Debug, Clone, Default)]
+pub struct VMBuilder {
```

Comment:
Should we name this `VMStateBuilder` to avoid any confussions?

Reply:
Yeah, that's a good idea.